tests/mapped_struct.py

```diff
@@ -1570,6 +1558,91 @@ def testIndexedCollectionEqual(self):
         self.assertNotEquals(c, pack([1, 2.0, 3]))
         self.assertNotEquals(c, pack([2, 2.0]))
 
+class ProxiedFrozensetPackingTest(unittest.TestCase, CommonCollectionPackingTest):
+    PACKING_CLASS = mapped_struct.proxied_frozenset
+    COLLECTION_CLASS = frozenset
+
+    def testContains(self):
+        values = [float(x) for x in range(5)]
+        c = self.pack(values)
+        for val in values:
+            self.assertIn(val, c)
+        for val in values:
+            self.assertNotIn(-val - 1, c)
+
+    def testCmpSets(self):
+        c = self.pack([1., 2., 3., 4., 5.])
+        self.assertEqual(c, self.pack([5., 4., 3., 2., 1.]))
+        self.assertEqual(c, set(c))
+        self.assertEqual(c, frozenset(c))
+
+    def testSetOps(self):
+        c = self.pack([1., 2., 3., 4., 5.])
+        self.assertEqual(c.union([3., 6.]), frozenset([1., 2., 3., 4., 5., 6.]))
+        self.assertEqual(c.intersection([3., 5., 9.]), frozenset([3., 5.]))
+        self.assertEqual(c.difference([2., 4., 8.]), frozenset([1., 3., 5.]))
+        self.assertEqual(c.symmetric_difference([1., 8.]), frozenset([2., 3., 4., 5., 8.]))
+
+    def testSubsetSuperset(self):
+        small = self.pack([1., 2.])
+        big = self.pack([1., 2., 3., 4.])
+        self.assertTrue(small < big)
+        self.assertTrue(small <= big)
+        self.assertFalse(small < small)
+        self.assertTrue(big > small)
+        self.assertTrue(big >= small)
+        self.assertFalse(big > big)
+
+        big = self.pack([1., 1.5, 2., 2.5, 3.])
+        self.assertTrue(small < big)
+        self.assertTrue(big > small)
+
+    def testCompressed(self):
+        small = self.pack([1, 2])
```

Comment:
Add big numbers to force usage of both bitmaps.

Reply:
This particular test does check for correctness when both operands use bitmaps.

Reply:
Yes, if I am right, you have two bitmaps in each compressed frozenset. I suggest adding numbers that force the usage these both bitmaps.

Reply:
Not really. The higher portion of the bitmap is 0 here and will be skipped by the optimization you added later. Run coverage if you prefer. You should add a set with higher (yet bitmap-able) values so that the higher portion of the bitmap gets used.

Reply:
Oh, you meant usage of both `bitmap_lo` and `bitmap_hi`. I'm dumb.